virtio-devices/src/block.rs

```diff
@@ -93,6 +93,14 @@ pub enum Error {
         /// The path of the disk image.
         path: PathBuf,
     },
+    #[error("disk image size is not a multiple of {}", SECTOR_SIZE)]
```

Comment:
Capitalisation (I know Rust style has these as lower cases - but a lot of our code predates that establishment.)